src/triage/component/audition/thresholding.py

```diff
@@ -21,6 +21,66 @@ def _of_metric(df, metric_filter):
     ]
 
 
+class ModelGroupChecker(object):
```

Comment:
I only see this object used once for the result of its one public method.

Classes are powerful, and I would certainly believe that we foresee extending this class into something that uses such features, like additional methods which will leverage the object state, or user extensibility through subclassing.

That said, is this really something that could just be a function? If that's the case, its signature would be the same as this class's construction signature, and its output would be the same as its one public method's output….

Reply:
I agree that this can begin life as a function.

I also don't like the name. TO me, 'checking' sounds like it is going to return true or false, or maybe throw an error if a condition isn't met. What it seems to be doing is *filtering*. It may not be similar enough to the ModelGroupThresholder purpose tom combine with it, but we can still use filter in the name somehow